### ISSUE TITLE
401 Unauthorized errora from beta component using rest_of_world region

### ISSUE THREAD

**Issue by Djelibeybi** [bimmerconnected/bimmer_connected#365]
Hey folks,

I'm getting the following error when Home Assistant starts, but not every time. I haven't yet determined how to reproduce it reliably:

```bash
Logger: bimmer_connected.account
Source: /usr/local/lib/python3.9/site-packages/bimmer_connected/account.py:231
First occurred: 9:42:11 AM (1 occurrences)
Last logged: 9:42:11 AM
401 Client Error: Unauthorized for url: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles?apptimezone=660.0&appDateTime=1637408530.215427&tireGuardMode=ENABLED

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/bimmer_connected/account.py", line 229, in send_request
    response.raise_for_status()
  File "/usr/local/lib/python3.9/site-packages/requests/models.py", line 953, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles?apptimezone=660.0&appDateTime=1637408530.215427&tireGuardMode=ENABLED
```

**Comment by Djelibeybi** [comment]
For completeness:

```log
2021-11-29 03:49:37 DEBUG (SyncWorker_5) [bimmer_connected.account] Request to: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles
2021-11-29 03:49:39 DEBUG (SyncWorker_5) [bimmer_connected.account] Request to: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles
2021-11-29 03:49:40 ERROR (SyncWorker_5) [bimmer_connected.account] 401 Client Error: Unauthorized for url: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles?apptimezone=660.0&appDateTime=1638078579.165597&tireGuardMode=ENABLED
requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles?apptimezone=660.0&appDateTime=1638078579.165597&tireGuardMode=ENABLED
2021-11-29 03:49:40 ERROR (SyncWorker_5) [custom_components.bmw_connected_drive] Could not connect to the BMW Connected Drive portal. The vehicle state could not be updated
2021-11-29 03:49:40 ERROR (SyncWorker_5) [custom_components.bmw_connected_drive] 401 Client Error: Unauthorized for url: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles?apptimezone=660.0&appDateTime=1638078579.165597&tireGuardMode=ENABLED
  File "/config/custom_components/bmw_connected_drive/__init__.py", line 319, in update
requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles?apptimezone=660.0&appDateTime=1638078579.165597&tireGuardMode=ENABLED
2021-11-29 03:54:37 DEBUG (SyncWorker_6) [custom_components.bmw_connected_drive] Updating vehicle state for account me@dje.li, notifying 12 listeners
2021-11-29 03:54:41 DEBUG (SyncWorker_6) [bimmer_connected.account] Request to: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles
2021-11-29 03:54:43 DEBUG (SyncWorker_6) [bimmer_connected.account] Request to: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles
```

**Comment by rikroe** [comment]
Closing as it was fixed within the library/custom component and is included in the HA 2021.12 beta, so it will be released in HA on Dec. 11th.
